libclamav/ole2_extract.c

```diff
@@ -1319,7 +1319,7 @@ static cl_error_t handler_enum(ole2_header_t *hdr, property_t *prop, const char
                         hwp_new = cli_calloc(1, sizeof(hwp5_header_t));
                         if (!(hwp_new)) {
                             status = CL_EMEM;
-                            break;
+                            goto done;
```

Comment:
This one is good though, don't want to keep going if we're out of memory.

But this could introduce a leak, where `hwp_check` isn't free'd.  We should initialize it to NULL, set it to NULL after we free it on line `1334`, and then free it after `done:` if it is not NULL.